yarn-project/noir-protocol-circuits/src/crates/types/src/abis/private_circuit_public_inputs.nr

```diff
@@ -51,10 +51,12 @@ struct PrivateCircuitPublicInputs {
     encrypted_log_preimages_length: Field,
     unencrypted_log_preimages_length: Field,
 
-    block_header: BlockHeader,
+    // Header of the block the transaction is executing against (not the block the transaction is included in).
+    historical_header: Header,
 
     contract_deployment_data: ContractDeploymentData,
 
+    // TODO(#4256): This is now in header. Nuke this.
```

Comment:
As mentioned earlier, these would end up being different from header when forks occur.

Reply:
Closed the issue and removed the comment